extract/extract/extractors/amazon.py

```diff
@@ -0,0 +1,321 @@
+    """
+    soup = parsed_page.beautiful_soup
+
+    name = soup.find("span", {"id": "productTitle"}).text.strip()
+    color = _get_color(soup)
+    size = _get_sizes(soup)
+    price = _get_price(parsed_page)
+    image_urls = _get_image_urls(soup)
+
+    currency = "EUR"
+
+    sustainability_spans = soup.find_all("span", id=re.compile("CPF-BTF-Certificate-Name"))
+    sustainability_texts = [span.text for span in sustainability_spans]
+    sustainability_labels = _sustainability_label_to_certificate(sustainability_texts)
+
+    brand = _get_brand(soup)
+    description = _get_description(soup)
+    asin = _find_from_details_section(soup, "ASIN")
+
+    return Product(
+        timestamp=parsed_page.scraped_page.timestamp,
+        url=parsed_page.scraped_page.url,
+        merchant=parsed_page.scraped_page.merchant,
+        category=parsed_page.scraped_page.category,
+        name=name,
+        description=description,
+        brand=brand,
+        sustainability_labels=sustainability_labels,
+        price=price,
+        currency=currency,
+        image_urls=image_urls,
+        color=color,
+        size=size,
+        gtin=None,
+        asin=asin,
+    )
+
+
+def _sustainability_label_to_certificate(labels: list[str]) -> Union[list, set]:
+    """
+    Helper function that extracts the sustainability information from the parsed HTML's label tag.
+
+    Args:
+        labels (str): Label string from the HTML span tag
+
+    Returns:
+        list[CertificateType]: List of `CertificateType` objects
+    """
+    sustainability_labels = load_and_get_sustainability_labels()
+    _LABEL_MAPPING = {
+        "Global Recycled Standard": CertificateType.GLOBAL_RECYCLED_STANDARD,
+        "Global Organic Textile Standard": CertificateType.GOTS_MADE_WITH_ORGANIC_MATERIALS,
+        "Organic Content Standard 100": CertificateType.ORGANIC_CONTENT_STANDARD_100,
+        "Organic Content Standard Blended": CertificateType.ORGANIC_CONTENT_STANDARD_BLENDED,
+        "Compact by Design (zertifiziert durch Amazon)": CertificateType.COMPACT_BY_DESIGN,
+        "Natrue": CertificateType.NATRUE,
+        "Cradle to Cradle Certified": CertificateType.CRADLE_TO_CRADLE,
+        "Responsible Wool Standard": CertificateType.RESPONSIBLE_WOOL_STANDARD,
+        "The Forest Stewardship Council": CertificateType.FOREST_STEWARDSHIP_COUNCIL,
+        "Das offizielle Nordische Umweltzeichen": CertificateType.NORDIC_SWAN_ECOLABEL,
+        "Reducing CO2": CertificateType.CARBON_TRUST_REDUCING,
+    }
+
+    result = {
+        CertificateType[certificate.split(":")[-1]]
+        for certificate, description in sustainability_labels.items()
+        if any(_get_matching_languages(description["languages"].values(), labels))
+    }
+
+    for label, certificate in _LABEL_MAPPING.items():
+        if label in labels:
+            result.update({certificate})
+
+    return result or {CertificateType.OTHER}
```

Comment:
use `[CertificateType.OTHER]` to keep the return type `List[str]`.

Reply:
BTW: Didn't understand how `and`/`or` operands work up until now! I would expect a bool return value here. However, found this one: https://docs.python.org/3/library/stdtypes.html#boolean-operations-and-or-not

Reply:
Resolved in https://github.com/calgo-lab/green-db/pull/65/commits/dbb01f2d15dc24e3d9905fca4032e323c57193fa